tutorials/generating-synthetic-data/synthetic-data-generation-6.md

```diff
@@ -1,225 +1,195 @@
 ---
 myst:
   html_meta:
-    keywords: "Inductiva API, Programming, HPC, Simulation"
-orphan: true
+    description: "This tutorial provides a step-by-step guide for benchmarking the synthetic data 
+    generation simulation using the Inductiva API. Learn how to configure and run benchmarks on various 
+    machine types from Google Cloud, analyze results, and visualize performance metrics for different 
+    computational setups."
+    keywords: "Inductiva API, Benchmarking, Performance, Metrics, Computation Time, Computation Cost,
+    Programming, HPC, Simulation, Tutorial, Synthetic Data Generation, Physics-ML, SPH"
 ---
 
-# Benchmark Computational Resources
+# Benchmarking Computational Resources
```

Comment:
@sarmento you'd have to start from this .md file, the other one was added as a draft for reference and this one is the one @HenriquePreto and I are working on.

Reply:
OK! Will do that. But can you please apply my comments to the right place.
Thank you!

Reply:
If applicable, of course...